lib/FilesHooks.php

```diff
@@ -570,11 +572,21 @@ protected function getUserPathsFromPath($path, $uidOwner) {
 		try {
 			$node = $this->rootFolder->getUserFolder($uidOwner)->get($path);
 		} catch (NotFoundException $e) {
-			return [];
+			$this->logger->warning('Something fishy happens: Path "{path}" with owner "{uidOwner}" was not found',
```

Comment:
It's a warning, no need to add such statement. :)
```suggestion
			$this->logger->warning('Path "{path}" with owner "{uidOwner}" was not found',
```